docs/help/account/billing.md

```diff
@@ -0,0 +1,44 @@
+# Billing
+
+The Billing section of your account provides access to the following items:
+
+- information about your current plan
+- your invoice history
+- your billing details
+
+Bump.sh offers monthly or annual billing. Payment is by credit card only; we do not offer any alternatives at this time. We use Stripe to handle all payments, so we never store your bank details.
+
+## Invoices
+
+### Edit Your Billing Information
```

Comment:
Why do we upcase each word? It's not something we used to do in the current doc version, is there any reason to change this? (Real question, I don't have any strong opinion on this)

Reply:
👍